rtb/norme_di_progetto_ver0.6.0.typ

```diff
@@ -531,9 +556,177 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+
+==== Riunioni
+Al fine di garantire l'efficienza delle riunioni il responsabile corrente avrà il compito di interloquire con i proponenti per riepilogare i punti principali del periodo trascorso e esprimere dubbi e incertezze emerse dal gruppo.
+
+===== Riunioni Interne
+Le riunioni interne sono programmate di comune accordo tra i membri del gruppo. In caso di necessità, è possibile richiedere riunioni straordinarie durante la settimana tramite il canale dedicato su Telegram, con data e orario stabiliti attraverso un sondaggio. Tutte le riunioni online si svolgeranno nel canale Discord appositamente designato. Nelle riunini interne l'obiettivo finale è quello di risolvere le mancanze segnalate dall'ordine del giorno, per cui nella fase finale della riunione, il responsabile, dopo aver discusso con il gruppo, pianifica le attività da svolgere e crea rispettivamente le _issue_ come da procedura sopra.
+
+===== Riunioni Esterne
+Le riunioni esterne coinvolgono i membri del gruppo e il proponente.\
+Per le riunioni con il proponente, viene utilizzata la piattaforma Google Meet, e l'indirizzo viene comunicato al _team_ di volta in volta. In accordo con l'azienda proponente gli incontri di fine sprint avvengono ogni due settimane, in quanto lo sprint dura due settimane, mentre a metà di ogni sprint si svolgerà un incontro di allineamento più breve per un confronto e degli aggiustamenti.
+
+===== Compiti del responsabile
+- Esporre i punti all'ordine del giorno;
+- Fare il punto su quanto fatto e sul rispetto degli obiettivi posti;
+- Pianificare le prossime attività da svolgere;
+- Assegnare i _task_ ai membri del gruppo.
+
+==== Verbali
+===== Verbali Interni
+Al fine di tracciare le discussioni svolte, le decisioni prese dal gruppo e soprattutto le motivazioni dietro ad esse, al termine di ogni incontro interno al gruppo, viene aperta una _issue_ su Github per la preparazione, la verifica e l'approvazione del verbale. Il compito di redigere il verbale, seguendo il formato indicato nella sezione 3.1 di questo documento, è affidato al responsabile, il quale deve assicurarsi di includere tutte le informazioni rilevanti discusse.
+
+===== Verbali Esterni
+Per quanto riguarda le riunioni con i proponenti, valgono le stesse regole di quelle interne, con il passaggio aggiuntivo, di cui si occupa il responsabile, di far firmare il verbale ai proponenti una volta redatto e approvato.
+
+== Miglioramento
+=== Descrizione
+Secondo lo standardG ISO/IEC 12207:1995, il processo di miglioramento nel ciclo di vita del software è finalizzato a stabilire, misurare, controllare e migliorare i processi che lo compongono. L’attività di miglioramento è composta da:
+- Analisi: Identificare le aree di miglioramento dei processi;
+- Miglioramento: Implementare le modifiche necessarie per migliorare i processi di sviluppo del software;
```

Comment:
- "software" da mettere in corsivo
- inoltre toglierei le maiuscole da "Identificare" e "Implementare"